src/hooks/useLauncherEventProvider/useLauncherEventProvider.ts

```diff
@@ -0,0 +1,136 @@
+import { useCallback, useState } from "react";
+import { sleep } from "../../lib/utils";
+import { log } from "../../lib/log";
+
+interface UseLauncherEventProviderEventsDelegate {
+  onInfoUpdates(
+    info: any
+  ): void;
+  onNewEvents(events: any): void;
```

Comment:
I don't love using `any` here but it's what overwolf sends

https://github.com/overwolf/types/blob/master/overwolf.d.ts#L2772-L2782